patches/master_patch.patch

```diff
@@ -160,6 +160,17 @@ index 10faefae1df8921bcdf0f7a43b0b3a279c171880..2b1a8ffee85bdbe3749a7e1be4619214
  #if BUILDFLAG(ENABLE_SPELLCHECK)
      registry_with_source_info_.AddInterface(
          base::BindRepeating(&SpellCheckHostChromeImpl::Create), ui_task_runner);
+@@ -130,8 +128,8 @@ class ChromeService::ExtraParts : public ChromeBrowserMainExtraParts {
+     // ChromeService::BindConnector() can race with ChromeService::OnStart().
+     ChromeService::GetInstance()->InitConnector();
+ 
+-    connection->GetConnector()->StartService(
+-        service_manager::Identity(chrome::mojom::kServiceName));
++    //connection->GetConnector()->StartService(
```

Comment:
this workaround took me a while to figure out the missing pieces of https://github.com/brave/muon/commit/237242113a697ab7647aa6a262b7ec887c37a2de